example/advection/redefine_parthenon_defaults.cpp

```diff
@@ -113,7 +115,8 @@ void Mesh::UserWorkAfterLoop(ParameterInput *pin, SimTime &tm) {
     IndexRange jb = pmb->cellbounds.GetBoundsJ(IndexDomain::interior);
     IndexRange kb = pmb->cellbounds.GetBoundsK(IndexDomain::interior);
 
-    // TODO(pgrete) needs to be a reduction when using parallel_for
+    // calculate error on host
+    auto q = rc.Get("advected").data.GetHostMirrorAndCopy();
```

Comment:
This is using the default pipeline, not the meshblock stream as you described in your README. Should I expose a meshblock version?

Reply:
See comment above. At some point this may be useful, but currently there aren't too many places (except for init and finalization where performance doesn't matter anyway/that much)